### PR TITLE
Move the scripts needed to generate build artifacts to `generate-build`

### PR DESCRIPTION
**Pull request by hughsie**
The `contrib` directory isn't the right place at all.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
Note, this needs the jcat fix copied to gusb too otherwise the mac-os build will fail too.

File: po/fix_translations.py
Comment (superm1) [inline po/fix_translations.py:1]: even though this is translations specific, I say move it to the new directory.  We could eventually aim for code duplication in that directory and have `common.py` that all the scripts import or something too.

Comment (hughsie) [reply po/fix_translations.py:1]: makes sense to me

Comment (superm1) [reply po/fix_translations.py:1]: To avoid having to bump subprojects now, can you leave a compat symlink for those projects?

Comment (hughsie) [reply po/fix_translations.py:1]: It's okay, we're overdue for tarballs anyway.

**Comment by hughsie** [comment]
@superm1 if you like the directory name then this works for me -- and then I can rebase the other PR on top.

**Comment by superm1** [comment]
> @superm1 if you like the directory name then this works for me -- and then I can rebase the other PR on top.

everything looks good to me, but maybe `generators` or `build-scripts` if those sound good to you?  I'm not too strongly feeling on the names.

**Comment by hughsie** [comment]
> maybe generators

I thought this too; but I couldn't get this working as actual meson generators and I think it might be confusing.

> or build-scripts

I wanted to avoid the build prefix -- it'll screw up my tab completion completely :)

**Comment by superm1** [comment]
> > maybe generators
> 
> I thought this too; but I couldn't get this working as actual meson generators and I think it might be confusing.
> 
> > or build-scripts
> 
> I wanted to avoid the build prefix -- it'll screw up my tab completion completely :)

Okay I don't feel strong.

**Pull request merged**